### ISSUE TITLE
BZIP2_INCLUDE_DIR variable not used

### ISSUE THREAD

**Issue by simonmssu** [python-cmake-buildsystem/python-cmake-buildsystem#103]
can we change python-cmake-buildsystem\cmake\extensions\CMakeLists.txt line 332 from

```
add_python_extension(bz2
    REQUIRES BZIP2_LIBRARIES
    SOURCES bz2module.c
    LIBRARIES ${BZIP2_LIBRARIES}
)
```

to

```
add_python_extension(bz2
    REQUIRES BZIP2_LIBRARIES BZIP2_INCLUDE_DIR
    SOURCES bz2module.c
    LIBRARIES ${BZIP2_LIBRARIES}
    INCLUDEDIRS ${BZIP2_INCLUDE_DIR}
)
```

so that we can specify bzip2 library installation directory?

I've also attached the [CMakeLists.txt](https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/files/40143/CMakeLists.txt) file

The reason is, I will need to turn on bz2 module in python to use the python in my qt5 compilation.

thanks
-s

**Comment by jcfr** [comment]
Thanks for the patch :+1: 

I created PR #104 

As soon as the continuous integration report back, I will integrate.

Jc

**Comment by jcfr** [comment]
I added you as author using `Simon Su <simonmssu@users.noreply.github.com>`. Let me know if you would like your real email.

**Comment by simonmssu** [comment]
Thanks JC, please use my simonmssu@gmail.com email address.


Thanks
-simon

On Saturday, November 21, 2015, Jean-Christophe Fillion-Robin <
notifications@github.com> wrote:

> I added you as author using Simon Su <simonmssu@users.noreply.github.com
> <javascript:_e(%7B%7D,'cvml','simonmssu@users.noreply.github.com');>>.
> Let me know if you would like your real email.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/issues/103#issuecomment-158594128>
> .
>


-- 
-simon

**Comment by jcfr** [comment]
Great. Patch integrated in 4197e6977

**Comment by simonmssu** [comment]
thank you much JC and thank you for making cmake for python.
-simon

On Tue, Nov 24, 2015 at 1:17 PM, Jean-Christophe Fillion-Robin <
notifications@github.com> wrote:

> Closed #103
> <https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/issues/103>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/issues/103#event-473618712>
> .
>



-- 
-simon
